capstone/src/main/java/com/google/univiz/SuggestionData.java

```diff
@@ -1,4 +1,18 @@
 package com.google.univiz;
 
+import com.google.auto.value.AutoValue;
+import com.google.gson.annotations.SerializedName;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+import javax.annotation.Nullable;
+
 /** SuggestionData represents the college data specific to the Search feature */
-public class SuggestionData {}
+@GenerateTypeAdapter
+@AutoValue
+abstract class SuggestionData {
+  @SerializedName("school.name")
+  @Nullable
```

Comment:
Why nullable? Returning null would make it difficult to handle errors when we want to show auto complete data in JS.

Reply:
Ah ok...what happens if I don't make it nullable-> would that throw an error right there and then? I was thinking I could check earlier up in the pipeline, but I can change that